### PR TITLE
[Preview] Tooltip position

### PR DESCRIPTION
**Pull request by DavideDeCenzo**
Associated Issue: #4583

Here's the Pull Request Doc
https://devtools-html.github.io/debugger.html/CONTRIBUTING.html#pull-requests

### Summary of Changes
Add block div in line 136/145  in src/components/shared/Popover.js

### Test Plan

Tested in https://event-handler.glitch.me/

Here's the Debugger's Testing doc
https://docs.google.com/document/d/1oBMRxV8A2ag2t22YsQOxTdEv0mXKzIg0tggJjRkU1S0/edit#.

### 
bug4583

### CODE REVIEW

**Comment by jasonLaster** [comment]
Looks good @DavideDeCenzo how did you fix it? What was the problem?

**Comment by DavideDeCenzo** [comment]
it missed a block div in renderPopover() in the return statement.  I 've discovered it  after some attempts in the code trying to change the  style

File: src/components/shared/Popover.js [hunk 1]
```diff
@@ -133,13 +133,15 @@ class Popover extends Component {
...
 
     return (
-      <div
-        className={classNames("popover", { up: orientation === "up" })}
-        onMouseLeave={this.onMouseLeave}
-        style={{ top, left }}
-      >
-        {arrow}
-        {this.getChildren()}
+      <div>
```
Comment (jasonLaster) [inline src/components/shared/Popover.js]: can we try and do this without an extra div?

Comment (DavideDeCenzo) [reply src/components/shared/Popover.js]: I'll try

**Comment by wldcordeiro** [comment]
Hey there @DavideDeCenzo I've looked over this and I have an idea that will work really well. 😄 

I'll lay it out here for you. Feel free to ask questions but I hope it gets you going. 

You're going to want to work in this file. https://github.com/devtools-html/debugger.html/blob/master/src/components/shared/BracketArrow.css

So basically we've been doing the arrow by using a border trick. This gets a little ugly though and is harder to position as we've seen. So my suggestion is replacing the borders with `▲` and `▼` then adjusting their colors and not border-colors.

```css
.bracket-arrow.up::before {
  content: "▲";
  top: -1px;
}

.bracket-arrow.down::before {
  content: "▼";
  top: 0px;
}
```

**Comment by DavideDeCenzo** [comment]
Thanks a lot @wldcordeiro , This is my first bug so every advice for me is helpful :) . I'll try your idea then

**Comment by DavideDeCenzo** [comment]
I've changed the arrow as @wldcordeiro said and deleted extra div in Popover.js then I've added some margin in Popover.css for to obtain the same result as before without extra div . The final result is this :

fixdefinitvocarotv-2 
fixdefinitocarot2v2

Added in Popover.css
--- 
```css
.popover .preview-popup {
   margin-left: -55px;
}

.popover .add-to-expression-bar {
   margin-left: -55px;
}
```
BracketArrow.css
---

```css
.bracket-arrow {
  position: absolute;
}

.bracket-arrow::before,
.bracket-arrow::after {

  height: 0;
  width: 0;
  position: absolute;
  color: var(--theme-splitter-color);
  font-size: 16px;
  text-shadow: 0.5px 0.5px;
  filter: contrast(90%); 

}


.bracket-arrow.up::before {
  content: "▲";
  top: -5px;
  
}


.theme-dark .bracket-arrow.up::before {
  color: var(--theme-body-color);
  
}

.bracket-arrow.up::after {
  color: var(--theme-body-background);
  top: 0px;
}


.bracket-arrow.down::before {
  content: "▼";
  top: 0px;
}


.theme-dark .bracket-arrow.down::before {
  color: var(--theme-body-color);
}

.bracket-arrow.down::after {
  color: var(--theme-body-background);
  top: -1px;
}
```
Deleted extra div in Popover.js
---
```javascript
renderPopover() {
    const { top, left, orientation, targetMid } = this.state;
    const arrow = this.getPopoverArrow(orientation, targetMid);

    return (
      <div
        className={classNames("popover", { up: orientation === "up" })}
        onMouseLeave={this.onMouseLeave}
        style={{ top, left}}
      >
        {arrow}
        {this.getChildren()}
      </div>
    );
  }
```

**Comment by codehag** [comment]
@DavideDeCenzo it looks like you haven't pushed the recent changes... can you push and we will take a look at it?

**Comment by DavideDeCenzo** [comment]
@jasonLaster The changes are ok or is there something to modify?

**Comment by codehag** [comment]
@DavideDeCenzo i think we might want to remove the changes that added the "▲", since it changes the look of the arrow. We also have less control over it, and the old way is more standard for creating popups with arrows. Also, if we could use the text-outline property, we have no control over where that outline appears, so we would need to use the zindex and position things that way, which could also be really hacky. I didn't realize that was the change last time when i quickly looked at the screenshot. if it is alright, can we revert the last commit? What you had at 0ec67a1 is sufficient for what we need. 

I can do the revert, or if you would like to try it, you would do the following:

```
git reset --hard 0ec67a1 
git push <repo> <branchname> -f
```

git reset hard will take you back to the previous commit and forget all changes. git push .... -f will force push

**Comment by DavideDeCenzo** [comment]
@codehag  I see, I'm gonna try the revert , thanks for the review

**Comment by DavideDeCenzo** [comment]
Ok I've reverted it I think the PR is ready @jasonLaster

**Comment by codehag** [comment]
Thanks @DavideDeCenzo!

**Pull request merged**